ext/dom/lexbor/lexbor/css/syntax/state_res.h

```diff
@@ -49,7 +49,7 @@ lxb_css_syntax_state_res_map[256] =
     lxb_css_syntax_state_hash, /* 0x23; '#'; Pound sign */
     lxb_css_syntax_state_delim, /* 0x24; '$'; Dollar sign */
     lxb_css_syntax_state_delim, /* 0x25; '%'; Percentage sign */
-    lxb_css_syntax_state_delim, /* 0x26; '&'; Commericial and */
+    lxb_css_syntax_state_delim, /* 0x26; '&'; Commercial and */
```

Comment:
```suggestion
    lxb_css_syntax_state_delim, /* 0x26; '&'; Ampersand */
```

Reply:
This should probably be fixed upstream first.